src/store/modules/events/index.ts

```diff
@@ -561,13 +562,13 @@ const module: Module<EventsModuleState, RootState> = {
      * @param {EventsModuleState} state
      * @param {string} projectId - project event is related to
      * @param {string} eventId - visited event
-     * @param {string} userId - user who visited event
+     * @param {User} user - user who visited event
      */
-    [MutationTypes.MARK_AS_VISITED](state, { projectId, eventId, userId }) {
+    [MutationTypes.MARK_AS_VISITED](state, { projectId, eventId, user }) {
       const key = getEventsListKey(projectId, eventId);
 
       const event = state.list[key];
-      const visitedBy = new Set([...(event.visitedBy ? event.visitedBy : []), userId]);
+      const visitedBy = new Set([...(event.visitedBy ? event.visitedBy : []), user]);
```

Comment:
```suggestion
      const visitedBy = new Set([...(event.visitedBy || []), user]);
```

А ты проверял, там не добавляется один и тот же пользователь?
Раньше то были строки и они по значению сравнивались, а теперь объекты, которые по ссылке. Возможно здесь надо через `find` проверять, есть ли пользователь и только тогда добавлять.

С другой стороны, так как "развидеть" ивент нельзя и удалять объекты не придется, то может и пофиг на это